tests/validation/tests/v3_api/test_create_ha.py

```diff
@@ -124,7 +133,7 @@ def valid_response():
         assert False, "check the logs in console for details"
 
     print_kubeconfig()
-    if RANCHER_HA_HARDENED:
+    if RANCHER_HA_HARDENED and RANCHER_LOCAL_CLUSTER_TYPE == "RKE":
```

Comment:
Should this check be RKE2

Reply:
Nope this is RKE1. `profile` is only set in this block (line 78):
```
        if RANCHER_LOCAL_CLUSTER_TYPE == "RKE":
            print("RKE cluster is provisioning for the local cluster")
            nodes = create_resources()
            if RANCHER_HA_HARDENED:
                profile = 'rke-cis-1.5'
                node_role = [["worker", "controlplane", "etcd"]]
                node_roles =[]
                for role in node_role:
                    node_roles.extend([role, role, role])
                prepare_hardened_nodes(nodes, profile, node_roles)
            config_path = create_rke_cluster_config(nodes)
            create_rke_cluster(config_path)
```